Potatso/AppInitializer.swift

```diff
@@ -37,7 +37,7 @@ class AppInitializer: NSObject, AppLifeCycleProtocol {
         DDLog.add(fileLogger)
 
         let logglyLogger = LogglyLogger() // Loggy Logger
-        logglyLogger.logglyKey = InfoInternal.shared.getLogglyAPIKey()
+        logglyLogger.logglyKey = "107d98a8-c275-4369-a881-b36d3564c9ef"
```

Comment:
Could you please revert back to this `logglyLogger.logglyKey = InfoInternal.shared.getLogglyAPIKey()`?